theano/sandbox/cuda/__init__.py

```diff
@@ -296,11 +296,12 @@ def dnn_available():
   return 1;
 }
 """
-            params = ["-l", "cudnn", "-I" + os.path.dirname(__file__)]
+            params = ["-l", "cudnn"]
+            params.extend(['-I"%s"' % os.path.dirname(__file__)])
             if config.dnn.include_path:
-                params.append("-I" + config.dnn.include_path)
+                params.extend(['-I"%s"' % config.dnn.include_path]) 
```

Comment:
OK. Thanks for pointing this out.
On my machine I did not have to fix this to make GPU work, so I was unaware of this file.